tests/grooves/test_equivalent_ribbed_groove.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+from matplotlib import pyplot as plt
+from numpy import pi, isclose, rad2deg
+
+from pyroll.core import EquivalentRibbedGroove
+
+
+def check(g):
+    plt.figure(dpi=300)
+    plt.axes().set_aspect("equal")
+    plt.plot(*g.contour_points.T)
+    plt.show()
+
+    assert isclose(g.r1, 0.2)
+    assert isclose(g.r3, 3.45)
+    assert isclose(g.usable_width, 13.6788)
+    assert isclose(g.depth, 5.5091)
```

Comment:
These checks are just for the given input parameters, add more checks for calculated measures